### ISSUE TITLE
support broccoli 1.x

### ISSUE THREAD

**Issue by joebartels** [salsify/broccoli-gzip#4]
https://github.com/broccolijs/broccoli/blob/master/docs/broccoli-1-0-plugin-api.md

```
InvalidNodeError: GzipFilter: The .read/.rebuild API is no longer supported as of 
Broccoli 1.0. Plugins must now derive from broccoli-plugin. 
https://github.com/broccolijs/broccoli/blob/master/docs/broccoli-1-0-plugin-api.md
used as input node to BroccoliMergeTrees
```

**Comment by dfreeman** [comment]
Knocking out #3 should take care of this as well. I'm hoping to have a chance to get to it sometime in the next couple weeks, but if anyone else gets to it in the meantime, so much the better.

**Comment by dfreeman** [comment]
This should be fixed in 0.3.0, which was just released.
